addons/setup/tokenexchange-manifests/spoke_clusterrolebinding.yaml

```diff
@@ -16,4 +16,10 @@ subjects:
 - kind: Group
   apiGroup: rbac.authorization.k8s.io
   name: {{ .Group }}
+- kind: User
+  apiGroup: rbac.authorization.k8s.io
+  name: {{ .UserHub }}
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: {{ .GroupHub }}
```

Comment:
1. I don't think these are required
2. Can you add a new line to the end of the file?